### ISSUE TITLE
Fix `ERESOLVE` error during default install via `npm@7.x`

### ISSUE THREAD

**Issue by rishihashbinary** [bkrem/react-d3-tree#347]
Hello Team, I was trying to install the library in my application built using nextjs and react 17.0, however seems like the react version supported for the library if 16.0.

Following is the error log whiule installing the library
````
$ npm i --save react-d3-tree
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR!   peer react@"16.x || 17.x" from react-d3-tree@2.0.1
npm ERR!   node_modules/react-d3-tree
npm ERR!     react-d3-tree@"*" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.0.0 || ^16.0.0" from react-transition-group@1.2.1
npm ERR! node_modules/react-d3-tree/node_modules/react-transition-group
npm ERR!   react-transition-group@"^1.1.3" from react-d3-tree@2.0.1
npm ERR!   node_modules/react-d3-tree
npm ERR!     react-d3-tree@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See C:\Users\rishi\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\rishi\AppData\Local\npm-cache\_logs\2021-06-03T09_09_35_849Z-debug.log
````
Thanks in advance.

**Comment by bkrem** [comment]
Hi @rishihashbinary,

Thanks for making me aware of this. This seems to be some stricter peer dependency handling introduced by `npm v7` (`npm install` for `npm v6` and `yarn add` work as expected).

The library supports `react@17.x` (see the line `peer react@"16.x || 17.x" from react-d3-tree@2.0.1` in the stack trace), but it seems to be failing on the `react-transition-group` dependency specified by the library itself.

**Please try running `npm --save --legacy-peer-deps react-d3-tree`, which will install the library as expected.**

I will leave this issue open for visibility until a solution is in place to handle this change in `npm` without requiring additional CLI flags.

**Comment by bkrem** [comment]
Fixed in [v2.0.2](https://github.com/bkrem/react-d3-tree/releases/tag/v2.0.2) 🚀
